### PR TITLE
dotCMS/core#12631

### PR DESCRIPTION
**Pull request by hmoreras**

### CODE REVIEW

File: src/app/view/components/main-navigation/main-navigation.component.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    private getMenuSelected(menuItemSelectedId: string): Menu {
+        return this.menuItems.filter(
+            menu => menu.menuItems.filter(menuItem => menuItem.id === menuItemSelectedId).length > 0
+        )[0];
+    }
+
+    public  setMenuActiveTabName(id?: string) {
+        this.open = !this.open;
+        this.menuItemIdActive  = id ? id : this.routingService.currentPortletId;
```
Comment (jdotcms) [inline src/app/view/components/main-navigation/main-navigation.component.ts]: I think you can use id | this.routingService.currentPortletId;

Comment (fmontes) [reply src/app/view/components/main-navigation/main-navigation.component.ts]: Can do:

```this.menuItemIdActive = id || this.routingService.currentPortletId;```

Comment (jdotcms) [reply src/app/view/components/main-navigation/main-navigation.component.ts]: Yeah it is exactly what I meant :)

File: src/app/view/components/main-navigation/main-navigation.component.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { Component, ViewEncapsulation, OnInit } from '@angular/core';
+import { RoutingService, Menu } from '../../../api/services/routing-service';
+
+@Component({
+    encapsulation: ViewEncapsulation.None,
+
```
Comment (fmontes) [inline src/app/view/components/main-navigation/main-navigation.component.ts]: extra line

File: src/app/view/components/main-navigation/main-navigation.component.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    providers: [],
+    selector: 'dot-main-nav',
+    styleUrls: ['./main-navigation.component.scss'],
+    templateUrl: 'main-navigation.component.html'
+})
+export class MainNavigationComponent implements OnInit {
+    private menuItems: Menu[];
+
```
Comment (fmontes) [inline src/app/view/components/main-navigation/main-navigation.component.ts]: extra line

**Pull request merged**